airflow/hooks/postgres_hook.py

```diff
@@ -1,77 +1,22 @@
 import psycopg2
 
-from airflow import settings
-from airflow.utils import AirflowException
-from airflow.models import Connection
+from airflow.hooks.dbapi_hook import DbApiHook
 
 
-class PostgresHook(object):
+class PostgresHook(DbApiHook):
     '''
     Interact with Postgres.
     '''
```

Comment:
The postgres hook seemed to be different than the other ones... I removed the ability to set a host/login/password/etc separately... Do you think it is necessary to have that?